source/includes/uri-usage-disclaimer.rst

```diff
@@ -1,4 +1,5 @@
 .. note::
 
-   URI's are optional, you can use connection string forms to
```

Comment:
(praise)
Nice rewrite! 

Consider also adding links to the Projects and Sync jobs pages.

Reply:
Thanks, I will leave the links off for now, it will be easier to add when the refs are in master and avoids merge conflicts for the projects drawer PR.